http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -52,10 +52,14 @@ public class Oidc {
     public static final String TEXT_CONTENT_TYPE = "text/*";
     public static final String DISCOVERY_PATH = ".well-known/openid-configuration";
     public static final String KEYCLOAK_REALMS_PATH = "realms/";
+    public static final String KEYSTORE_PASS = "password";
+    public static final String PKCS12_KEYSTORE_TYPE = "PKCS12";
     public static final String JSON_CONFIG_CONTEXT_PARAM = "org.wildfly.security.http.oidc.json.config";
     static final String ACCOUNT_PATH = "account";
     public static final String CLIENTS_MANAGEMENT_REGISTER_NODE_PATH = "clients-managements/register-node";
     public static final String CLIENTS_MANAGEMENT_UNREGISTER_NODE_PATH = "clients-managements/unregister-node";
+    public static final String ADMIN_CONSOLE_PATH = "admin/master/console/#";
```

Comment:
Please check all the new constants introduced in this class, it looks like these were meant for tests purposes.

Reply:
removed